handles.js

```diff
@@ -0,0 +1,73 @@
+/* jshint esversion:6*/
+const handles = function () {
+  return {
+    getHandle : getHandle,
+    postHandle : postHandle,
+    postReset : postReset,
+    putHandle : putHandle,
+    deleteHandle : deleteHandle,
+
+  };
+};
+
+let buzzArr = [];
+let score = 0;
+
+function getHandle (req, res){
+  let buzzwords = buzzArr.map( (obj) => {
+    return obj.buzzword;
+  });
+  let buzzObj = {buzzwords};
+  res.send(buzzObj);
+}
+
+function postHandle (req, res){
+  let matches = req.body.buzzword.match(/\d+/g);
+  if (isNaN(req.body.points) || matches != null){
```

Comment:
isNaN is probably not the best check to see if points is a number or not. Also, using the regex for buzzword is probably overthinking the problem again.